pkg/services/folder/folderimpl/folder.go

```diff
@@ -247,6 +249,10 @@ func (s *Service) Get(ctx context.Context, q *folder.GetFolderQuery) (*folder.Fo
 	f.ID = dashFolder.ID
 	f.Version = dashFolder.Version
 
+	if !q.WithFullpath {
+		f.Fullpath = f.Title // set full path to the folder title (unescaped)
+	}
```

Comment:
semantically, this is a bit different. GetFolders populates this field only if it was requested but sets to title if `nestedFolders` flag is disabled